.circleci/config.yml

```diff
@@ -144,6 +156,13 @@ jobs:
       - deploy_to_env:
           env: "preprod"
 
+  # deploy_prod:
```

Comment:
can this be deleted?

Reply:
It's not got a prod deployment yet but thought I'd add this is so when it does get one - it's just a case of uncommenting it, and the slack notification param is there ready.